### ISSUE TITLE
support for non-hashable objects

### ISSUE THREAD

**Issue by stared** [mouradmourafiq/pandas-summary#4]
If a column has non-hashable objects, creation of DataFrameSummary gives an error:

    TypeError: unhashable type: 'list'

It would be great if work also for these entries (e.g. looking at their length); or at very least - discard these variables.

**Comment by dsalaj** [comment]
Exactly. It would be much more usable if it could at least ignore unhashable type columns. Or even better count the number of empty rows. That would be very useful.

**Comment by mouradmourafiq** [comment]
PRs are welcome :)
